data/config/param_item_modifiers.yaml

```diff
@@ -44,3 +44,8 @@ META:
       required: false
       description: One or more values to be stored for this Meta Name
       values: .*
+OVERLAP:
+  summary: This item is allowed to overlap the previous item
```

Comment:
This should really say any earlier item in the packet.

Reply:
It now says "This item is allowed to overlap other items in the packet"